crates/parcel/src/request_tracker/mod.rs

```diff
@@ -1,6 +1,6 @@
 mod request;
 mod request_graph;
-mod request_tracker;
+pub mod request_tracker;
```

Comment:
How come it's public? I thought everything should go through the `parcel` function or whatever that will be

Reply:
I'm not 100% actually, I inherited this from the changes Pedro made. I'll see if I can make it private again.